docs/building/windows.md

```diff
@@ -0,0 +1,60 @@
+# How-To: Build Alephium `desktop-wallet` in Windows (10)
+
+This document is written with the intention of demonstrating the build process in Windows.
+
+**Note**: This guide was written for Windows 10, however the process should be the same for Windows 11. Please make an issue if you come across a Windows 11 specific issue and it will be addressed.
+
+## Getting Started / Prerequisites
+
+### Instal NodeJS
+
+Navigate to the [NodeJS Offical Installer](https://nodejs.org/en/download/) and download the Windows installer proper for your architecture (the most common choice is `x64` in `.msi` format)
+
+After the download completes, open the installer and complete the wizard (all defaults are fine)
+
+
+## Building
+
+
+### Clone the Project
+
+Download the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet).
+
+This can be accomplished by selecting the green `<>Code` button and then `Download ZIP` from the dropdown.
+
+Or by following this [link](https://github.com/alephium/desktop-wallet/archive/refs/heads/master.zip).
+
+
+### Navigate to the Project Within a Terminal
+
+Open a new terminal within the local repo with the following steps:
+
+1. Open `File Explorer`
+
+2. Navigate to, and open, the repo folder.
```

Comment:
For what it's worth I recall you can directly go into zip files on Windows. Will everything still "just work" or should they first extract the zip?

Reply:
You are correct! I should include the step to extract the files!